# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webrtc/RTCPeerConnection-iceConnectionState.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/webrtc/RTCPeerConnection-iceConnectionState.https.html.ini
--- a/testing/web-platform/meta/webrtc/RTCPeerConnection-iceConnectionState.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/webrtc/RTCPeerConnection-iceConnectionState.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,73 +1,74 @@
 [RTCPeerConnection-iceConnectionState.https.html]
   expected:
     if (os == "linux") and webrender and not fission: ["OK", "ERROR"]
-    if (os == "mac") and not debug: ["OK", "TIMEOUT"]
+    if (os == "mac") and not debug: ["TIMEOUT", "OK"]
     if (os == "mac") and debug: ["OK", "TIMEOUT"]
     if (os == "win") and webrender: ["OK", "TIMEOUT"]
   [connection with one data channel should eventually have connected connection state]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1278299
     expected:
+      if (os == "mac") and debug: ["FAIL", "TIMEOUT"]
+      if (os == "mac") and not debug: ["TIMEOUT", "FAIL"]
       if (os == "win") and webrender: ["FAIL", "TIMEOUT"]
-      if os == "mac": ["FAIL", "TIMEOUT"]
       FAIL
 
   [iceConnectionState changes at the right time, with bundle policy max-bundle]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1307996
     expected:
-      if (os == "mac") and not debug: ["FAIL", "NOTRUN"]
+      if (os == "win") and webrender: ["FAIL", "NOTRUN"]
+      if (os == "mac") and not debug: ["NOTRUN", "FAIL"]
       if (os == "mac") and debug: ["FAIL", "NOTRUN"]
-      if (os == "win") and webrender: ["FAIL", "NOTRUN"]
       FAIL
 
   [iceConnectionState changes at the right time, with bundle policy max-compat]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1307996
     expected:
-      if (os == "mac") and not debug: ["FAIL", "NOTRUN"]
+      if (os == "win") and webrender: ["FAIL", "NOTRUN"]
+      if (os == "mac") and not debug: ["NOTRUN", "FAIL"]
       if (os == "mac") and debug: ["FAIL", "NOTRUN"]
-      if (os == "win") and webrender: ["FAIL", "NOTRUN"]
       FAIL
 
   [iceConnectionState changes at the right time, with bundle policy balanced]
     bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1307996
     expected:
       if (os == "win") and webrender: ["FAIL", "NOTRUN"]
+      if (os == "mac") and not debug: ["NOTRUN", "FAIL"]
       if (os == "mac") and debug: ["FAIL", "NOTRUN"]
-      if (os == "mac") and not debug: ["FAIL", "NOTRUN"]
       FAIL
 
   [Closing a PeerConnection should not fire iceconnectionstatechange event]
     expected:
       if (os == "win") and webrender: ["PASS", "NOTRUN"]
       if (os == "mac") and debug: ["PASS", "NOTRUN"]
-      if (os == "mac") and not debug: ["PASS", "NOTRUN"]
+      if (os == "mac") and not debug: ["NOTRUN", "PASS"]
 
   [ICE can connect in a recvonly usecase]
     expected:
+      if (os == "win") and webrender: ["PASS", "NOTRUN"]
       if (os == "mac") and debug: ["PASS", "NOTRUN"]
-      if (os == "mac") and not debug: ["PASS", "NOTRUN"]
-      if (os == "win") and webrender: ["PASS", "NOTRUN"]
+      if (os == "mac") and not debug: ["NOTRUN", "PASS"]
 
   [connection with audio and video tracks should eventually have connected connection state]
     expected:
       if (os == "win") and webrender: ["PASS", "NOTRUN"]
+      if (os == "mac") and not debug: ["NOTRUN", "PASS"]
       if (os == "mac") and debug: ["PASS", "NOTRUN"]
-      if (os == "mac") and not debug: ["PASS", "NOTRUN"]
 
   [connection with audio track should eventually have connected connection state]
     expected:
+      if (os == "win") and webrender: ["PASS", "TIMEOUT"]
       if (os == "mac") and debug: ["PASS", "TIMEOUT"]
-      if (os == "mac") and not debug: ["PASS", "TIMEOUT"]
-      if (os == "win") and webrender: ["PASS", "TIMEOUT"]
+      if (os == "mac") and not debug: ["TIMEOUT", "PASS"]
 
   [Responder ICE connection state behaves as expected]
     expected:
       if (os == "win") and webrender: ["PASS", "NOTRUN"]
-      if (os == "mac") and not debug: ["PASS", "NOTRUN"]
       if (os == "mac") and debug: ["PASS", "NOTRUN"]
+      if (os == "mac") and not debug: ["NOTRUN", "PASS"]
 
   [connection with one data channel should eventually have connected or completed connection state]
     expected:
+      if (os == "win") and webrender: ["PASS", "TIMEOUT"]
+      if (os == "mac") and not debug: ["TIMEOUT", "PASS", "FAIL"]
       if (os == "mac") and debug: ["PASS", "FAIL", "TIMEOUT"]
-      if (os == "mac") and not debug: ["PASS", "FAIL", "TIMEOUT"]
-      if (os == "win") and webrender: ["PASS", "TIMEOUT"]
 